packages/askui-nodejs/example_projects_templates/typescript_jest/test/helper/jest.setup.ts

```diff
@@ -10,12 +12,29 @@ let aui: UiControlClient;
 jest.setTimeout(60 * 1000 * 60);
 
 beforeAll(async () => {
+  const proxyAgents = {};
+  /*
+  // Proxy configuration
+  // Uncomment this block for proxy support.
```

Comment:
I would move these comments above the block comment so that you don't have comment in comment.

Reply:
Build proxy agents from env variables

Reply:
and removed this code block